### DISCUSSION TITLE
add hidden cursor feature to autopilot mod

### DISCUSSION THREAD

**Discussion by Nepp0** [ppy/osu#23526] [Ideas]
autopilot cursor can be distracting, i use the no scope mod and it's visually fun to play with autopilot, but i don't think they should be compatible because autopilot aims at circles anyway whether the cursor is hidden or not. adding an option to hide the autopilot cursor would be cool.

**Comment by Gyoshi** [comment]
You have an option to hide the cursor. I don't understand what is actually the problem with no scope + autopilot...

**Reply by Nepp0** [reply]
i just don't think that's how no scope was intended to be used as a mod. i only use it because of the visual eye candy when playing with autopilot, and this effect can be achieved without the need to select 2 mods, that's all.
